src/breadcrumbs/_base.scss

```diff
@@ -2,40 +2,29 @@
  * @file Breadcrumbs
  * @module breadcrumbs
  * @overview Styles for Casper breadcrumbs
- * @TODO: expand/solidify styling, here they have styles to support grey and blue (like on the main site)
+ * @TODO: create set for footer breadcrumbs
```

Comment:
Is the footer breadcrumb code that unique? Let's sit down together and look at this! I'm still around if you want to do that now. Or tomorrow AM.